src/Symfony/Component/Serializer/Encoder/CsvEncoder.php

```diff
@@ -132,7 +132,7 @@ public function supportsEncoding(string $format): bool
     /**
      * {@inheritdoc}
      */
-    public function decode(string $data, string $format, array $context = []): mixed
+    public function decode(string $data, string $format, array $context = [])
```

Comment:
does apiplatform extend CsvEncoder?
if not, this change is not needed isn't it?
can you please consider the same question for all concrete classes?

Reply:
We don't. Let’s keep the types for concrete classes we don't extend.